tests/test_odgovlt.py

```diff
@@ -195,3 +199,18 @@ def test_OdgovltHarvester(app, db, mocker):
     assert result['report_status'] == 'added'
     assert result['errors'] == []
     assert was_last_job_considered_error_free()
+    ckanapi = CkanAPI()
+    ids = []
+    for x in ckanapi.package_list():
+        ids.append(x)
+    tags = ckanapi.package_show(id=ids[0])['tags']
+    assert sorted([x['name'] for x in tags]) == [
+                       'eismo intensyvumas',
+                       'keliai']
+    tags = ckanapi.package_show(id=ids[1])['tags']
+    assert sorted([x['name'] for x in tags]) == [
+                       'licencijos',
+                       'licencijuojamos veiklos teritorija',
+                       'šiluma',
+                       'šilumos tiekimo licencijas turinčių įmonių sąrašas',
+                       'šilumos tiekėjai']
```

Comment:
Now you test only tags, what about package name, description and other this you import?